src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -706,33 +728,19 @@ protected function getUTCDateTime(\DateTime $dateTime)
     /**
      * Removes all HTML markup & decode HTML entities
      *
+     * @deprecated This method is deprecated since KunstmaanNodeSearchBundle 5.7 and will be removed in KunstmaanNodeSearchBundle 6.0. Use the "removeHtml" method of the "Kunstmaan\NodeSearchBundle\Services\SearchViewRenderer" instead.
+     *
      * @param $text
      *
      * @return string
      */
     protected function removeHtml($text)
     {
-        if (!trim($text)) {
-            return '';
-        }
-
-        // Remove Styles and Scripts
-        $crawler = new Crawler();
-        $crawler->addHtmlContent($text);
-        $crawler->filter('style, script')->each(function (Crawler $crawler) {
-            foreach ($crawler as $node) {
-                $node->parentNode->removeChild($node);
-            }
-        });
-        $text = $crawler->html();
-
-        // Remove HTML markup
-        $result = strip_tags($text);
+        @trigger_error(sprintf('The "%s" method is deprecated since KunstmaanNodeSearchBundle 5.7 and will be removed in KunstmaanNodeSearchBundle 6.0. Use the "removeHtml" method of the "%s" service instead.', __METHOD__, SearchViewRenderer::class), E_USER_DEPRECATED);
 
-        // Decode HTML entities
-        $result = trim(html_entity_decode($result, ENT_QUOTES));
+        $searchViewRenderer = $this->container->get('kunstmaan_node_search.service.search_view_renderer');
 
-        return $result;
+        return $searchViewRenderer->removeHtml($text);
```

Comment:
is there a reason called the searchViewRenderer service only for the removeHTML call and not for the other 2?

Reply:
I think the reason is if someone had only overwritten the removeHTML call then their changes would break on upgrade because it would now be called somewhere else..

Reply:
Yes because fro the methods in the class renderDefaultSearchView and renderCustomSearchView we need to call those methods in case they are overriden by the user otherwise we can just call the new code (this way we won't trigger deprecations ourself

Reply:
We still call `$this->removeHtml()` in the old methods and if users override those render methods they still will can `$this->removeHtml()` so it would end up in their override or in the method and it will use the new class